src/main/java/org/folio/fqm/service/CrossTenantQueryService.java

```diff
@@ -22,39 +23,61 @@ public class CrossTenantQueryService {
 
   private final SimpleHttpClient ecsClient;
   private final FolioExecutionContext executionContext;
+  private final EntityTypeFlatteningService entityTypeFlatteningService;
+  private final PermissionsService permissionsService;
 
   public List<String> getTenantsToQuery(UUID entityTypeId) {
     String centralTenantId;
     String consortiumId;
+    List<HashMap<String, String>> userTenantHashMaps;
```

Comment:
Nit: The fact that this uses `HashMap` is an implementation detail that isn't particularly important, so I wouldn't mention it in the variable name. I'd lean toward something more like `userTenantMaps`

Nit #2: Can we change the type on this to `List<Map<String, String>>`?

Nit #3 (maybe not a nit, since I care more about this one): It's not obvious what this variable is or what it contains (something about users and tenants, but I'm not sure what each map is or what the keys and values are). A short comment or something would go a long way here